packages/push/src/instruction/bool.rs

```diff
@@ -50,8 +50,10 @@ where
     //   - result goes on the char stack
     let transaction: Transaction<PushState> = state.transaction();
 
-    let string = transaction.try_pop<String>()?; // This version has the transaction be mutable.
-    let (string, transaction) = transaction.try_pop<String>()?; // This version returns a "new" transaction.
+    let string = transaction.try_pop<String>()?;
+    // This version has the transaction be mutable.
+    let (string, transaction) = transaction.try_pop<String>()?;
+    // This version returns a "new" transaction.
```

Comment:
```suggestion
    // This version has the transaction be mutable.
    let string = transaction.try_pop<String>()?;
    // This version returns a "new" transaction.
    let (string, transaction) = transaction.try_pop<String>()?;
```

This is all in a big commented out block, so it doesn't _really_ matter, but what `cargo fmt` did was put the comments _after_ the lines they were annotating, which was at best confusing.

Reply:
Oh yea this is not cargo fmt's fault but mine xD